tests/test_modern.py

```diff
@@ -30,6 +30,13 @@
 )
 
 
+# Decorator for tests that require git
+skip_if_no_git = pytest.mark.skipif(
+    not pyclean.cli.is_git_available(),
+    reason='Git not installed'
+)
```

Comment:
This is repeated here. Not good. If you move this code to `conftest.py`, will the decorator continue to work?